stubs/watchpoints/watchpoints/__init__.pyi

```diff
@@ -0,0 +1,11 @@
+from collections.abc import Callable
+from typing import Final, Literal
+from typing_extensions import LiteralString, Unpack
+
+from .watch import Watch
+
+__version__: Final[LiteralString]
+
+all: Final[list[Literal["watch", "unwatch"]]]
```

Comment:
Fixed in https://github.com/python/typeshed/pull/13248/commits/7ae5b5cc68bbac71bb289d79ce082f972b1a977c - [PR is also merged in the runtime package](https://github.com/gaogaotiantian/watchpoints/pull/39) to change it to `__all__`, but I think that needs a new release before it can be added to typeshed.